src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,35 @@
+import ArticleListCard from "./ArticleListCard";
+import { getArticles } from "./utils/utils";
+import { useState, useEffect } from "react";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState("");
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+  useEffect(() => {
+    getArticles()
+      .then((data) => {
+        setArticles(data);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setIsError(true);
+      });
+  }, []);
+
+  if (isError) {
+    return <p>Error!!!</p>;
```

Comment:
Maybe instead of 'Error!!!' you could display a little more user-friendly message, something like: 'Failed to load articles'.